### DISCUSSION TITLE
transcribe error - index -1 out of bounds for length 0

### DISCUSSION THREAD

**Discussion by aum7** [Audiveris/audiveris#767] [Q&A]
attached file gives me error:
index -1 out of bounds for length 0

i tried scaling down (shrink) the image,
then i got '... with 6 pixel between ... try enlarge image resolution ...'
so the size of original file should be ok

i also tried cleaning / processing the image(s) with
- unsharp mask
- then remove grays
i get pretty good resulting images, but same error appears

while resolution might be quite big,
in past audiveris (ole! lol) showed popup notification saying 'file to big ...'

original image - 1563 x 2207 px
![img-011](https://github.com/user-attachments/assets/a5ada745-f5b3-43c7-87bc-47a067809ebd)

processed image - 1563 x 2207 px
![img-011](https://github.com/user-attachments/assets/ef899c80-d141-4b0e-99ec-d06e18ba381d)

this is the only image(s) i have trouble with
the rest of my audiveris (ole!) adventure has been a smooth sailing

EDIT: almost complete image is otherwise correctly transcribed / detected
if i export .mxl (or .xml)
and import it into musescore
i get completely blank / empty page

![audiveris indexoutofbound](https://github.com/user-attachments/assets/11aa86e9-8e9e-4c2e-b62f-cfe0f1214236)

**Comment by aum7** [comment]
cleaned the image a bit more
now it works as expected

seems top staff
with undefined barlines & time signature
is the root of all evil on this planet (lol, kidding)

![img-011_removed_imgs](https://github.com/user-attachments/assets/43af23b4-b68c-49e8-8719-be776b5f0e47)

EDITh (piaf lol): the other thing i removed manually are side and bottom black lines (probably remnants of scanned book pages)

**Comment by aum7** [comment]
is there an option to select an area of the image and mark it as unworthy of transcribing ?
tried & searched, could not find

EDIT: of course with [ctrl] for multiple areas selection

**Reply by hbitteur** [reply]
> is there an option to select an area of the image and mark it as unworthy of transcribing ?
> tried & searched, could not find

This feature does not exist yet.
